src/Components/LandingPage/Terminal/terminal.css

```diff
@@ -72,6 +72,8 @@
     border-top-right-radius: 20px;
     display: flex;
     padding-left: 10px;
+    cursor: default;
+    -webkit-user-select: none; 
```

Comment:
Adding only vendor prefixed version is a bad idea.We are already using autoprefixer, which will take care of all the prefixing necessary. just add the root style.

Reply:
User select is a non standard css property, meaning it is mandatory for it to have vendor prefix. Now that I'm aware of you guys using autoprefixer, I think we're good without it. You can check my commit.

Reply:
I meant use all vendor prefixes and the root style. such as -moz-user-select and so on